### PR TITLE
refactor: [M3-6245] - MUI v5 Migration - Features > Billing

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
Migrates `SRC > Features > Billing` from JSS to tss-react (Emotion)

## Major Changes 🔄 
- `UpdateContactInformationForm.tsx`: Showed how to convert class components using `withStyles.getClasses()`.
- Removed `@mui/styles` imports
- Removed `React.FC`
- Removed deprecated `createStyles` and unnecessary `withStyles`
- Other minor cleanup

## How to test 🧪
- Go to `/billing/` locally and ensure all functionality still works

### CODE REVIEW

File: packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentInformation.tsx [hunk 1]
```diff
@@ -6,81 +6,27 @@ import PayPalIcon from 'src/assets/icons/payment/payPal.svg';
...
 import DeletePaymentMethodDialog from 'src/components/PaymentMethodRow/DeletePaymentMethodDialog';
-import styled from 'src/containers/SummaryPanels.styles';
 import PaymentMethods from 'src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentMethods';
 import { queryKey } from 'src/queries/accountPayment';
 import { queryClient } from 'src/queries/base';
 import { getAPIErrorOrDefault } from 'src/utilities/errorUtils';
 import AddPaymentMethodDrawer from './AddPaymentMethodDrawer';
 
-const useStyles = makeStyles((theme: Theme) => ({
-  ...styled(theme),
```
Comment (jaalah-akamai) [inline packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentInformation.tsx:24]: You can't spread styles like this anymore. Most of the styles here were unused as well.

**Comment by jaalah-akamai** [comment]
> I'm noticing some visual differences between http://localhost:3000/account/billing and https://cloud.linode.com/account/billing when I compare them

Good catch @bnussman-akamai I had to update the base components used for `Billing` so that the styles applied correctly. We're going to switch to tackling all the base `/components/` moving forward. 👍 

Let me know if you notice any other differences and I'll address them!

**Comment by jaalah-akamai** [comment]
> LGTM aside from the spacing and positioning differences alluded to previously.
> 
> Is the intent here to get this merged and the differences reconciled through the relevant `/components` PRs?

Yea the `/component/` pr's will fix things, but maybe I can address them now. What specifically are you noticing?

**Comment by dwiley-akamai** [comment]
> Yea the `/component/` pr's will fix things, but maybe I can address them now. What specifically are you noticing?

Nothing visually broken, but some spacing narrowed, and if you open up this branch in one tab and prod in another and cycle back and forth between them, it looks like some slight shifting took place (could be an optical illusion due to the space narrowing).

<details>
<summary>Screenshots</summary>

**Prod**:
Screenshot 2023-03-21 at 1 24 59 PM

**This branch:**
Screenshot 2023-03-21 at 1 25 21 PM

</details>

**Review by dwiley-akamai — APPROVED** [review]

**Comment by jaalah-akamai** [comment]
@bnussman-akamai The gap between the cards is going to be slightly less because we'll have to convert the Grid component. That's something I'd rather do in a separate PR, but I'll make a note to circle back to the Billing page when it's done. I'll actually start working on the Grid next. Is that good?

**Review by bnussman-akamai — APPROVED** [review]
Sounds good! Sorry again about the `<Grid />`

**Pull request merged**